.golangci.yml

```diff
@@ -1,9 +1,14 @@
 run:
-  timeout: 2m
+  timeout: 5m
   tests: true
   skip-dirs:
+    - cmd/backfill_test
```

Comment:
I think this was inadvertently copied from sfxinternalgo.

Reply:
ack